README.md

```diff
@@ -49,7 +49,7 @@ Typing into that search field will result in a form submission (i.e. a
 Typically the server would respond with HTML like the following:
 
 ```html
-<ol>
+<ol aria-label="2 search suggestions">
```

Comment:
Should these ARIA labels perhaps be generated on the client side? I'm genuinely unsure whether this should be the server's job; what's the impact when used outside the context of this client-side component?

I'm also unsure whether this might be insufficiently obvious for server-side authors to respect. Plus there's the question of backwards compatibility: AIUI, existing server-side implementations would need to be updated with the current approach?